HISTORY.md

```diff
@@ -1386,7 +1386,7 @@
 	- [Chai](http://chaijs.com/) 1.1.x to 1.3.x
 - Removed ability to require uncompiled plugins
 	- This is due to the CoffeeScript 1.4.x from 1.3.x upgrade
-- `skeletonNonexistant` now tells us the path it checked
+- `skeletonNonexistent` now tells us the path it checked
```

Comment:
Is this a USA vs UK thing, or is that a universal thing?

Regardless, a change here will need to be a seperate pull requests that also addresses it in the code.